prow/plugins.yaml

```diff
@@ -67,6 +67,10 @@ plugins:
   kubernetes/kops:
   - trigger
 
+  kubernetes/kubectl:
+  - trigger
```

Comment:
It doesn't look like there are any jobs configured for kubernetes/kubectl in `prow/config.yaml` so trigger won't have any jobs to trigger for this repo.

Reply:
Cool, I'll just remove. We use travis for now, I'll revisit that when we/if we migrate.